.github/workflows/docs/conf.py

```diff
@@ -3,18 +3,18 @@
 # Configuration file for the Sphinx documentation builder.
 # See https://www.sphinx-doc.org/en/master/usage/configuration.html
 
-author = "Cambridge Quantum Computing Ltd"
+copyright = "2023 Quantinuum"
```

Comment:
Do we need to update this for the other extensions, too?

Reply:
I guess this line is a bit redundant as in the old extensions docs build the copyright info is manually added to the page [here](https://github.com/CQCL/pytket-qiskit/blob/develop/.github/workflows/docs/build-docs#L72).

In the tket repo the copyright info is only specifed in conf.py

I have updated pytket-quantinuum already

I think it'd be worth updating the other extensions too but wouldn't say this is urgent.

Reply:
I've removed the lines from `build-docs` so that the copyright info is only in `conf.py`.